### DISCUSSION TITLE
Is there plans to support echoes of the eye dlc?

### DISCUSSION THREAD

**Discussion by YoYoNinjaBoy** [Raicuparta/nomai-vr#426] [Q&A] [answered]
Title

**Comment by Raicuparta** [comment, marked as answer]
Yes, it will eventually be supported. There's just no telling how long it will take.

There was one "small" update they did once which didn't really have any visible changes for most users, and it took me a week of constant development to make NomaiVR support that version. So I imagine that a whole new DLC has the potential for much more complicated changes.

**Reply by YoYoNinjaBoy** [reply]
Awesome!  Thank you so much.  Nomai vr is an incredible experience.
